src/dune_file_watcher/dune_file_watcher.ml

```diff
@@ -575,6 +587,88 @@ let create_fsevents ?(latency = 0.2) ~(scheduler : Scheduler.t) () =
   ; sync_table
   }
 
+let winwatch_patterns =
+  List.rev_map ~f:Re.Str.regexp_case_fold
+    [ {|^_opam|}
```

Comment:
Some of this list is already present in `exclude_patterns` in this file.

Reply:
Moreover, there's duplication within this list. I'd like to see this regex defined in a non redundant way.